.github/workflows/publish-docker.yml

```diff
@@ -36,6 +42,7 @@ jobs:
         uses: docker/build-push-action@v2.9.0
         with:
           context: .
+          platforms: linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64/v8
```

Comment:
I sometimes see folks do this with `strategy` and `matrix`. Does that have any advantages like running in parallel compared to doing this way? I am not familiar with the build-push-action detials.

Reply:
Not an expert either, but I believe the outcome is the main difference
between both approaches and they tackle two different needs.

In my approach only one Docker image is generated with all the platforms,
allowing for easier distribution. Because of this, the name of the image
and tag is not changed from the current ones. Building for the different
platforms is done in parallel, but in the same runner.

With the matrix approach you'd get one image per platform. This of course
generates smaller images, but each user needs to pull the specific tag for
their platform, making distribution a bit harder. Building for the
different platforms is done in parallel, I think in different runners.

Both approaches I believe should work together. Most of the users first
download the multi platform build and might then switch to the platform
specific image if want to have a smaller image.

I implemented only the first one as it is the one I know how to implement
and it’s the less “disruptive”. I can try implementing also the matrix
approach, but can’t promise anything 😅

Reply:
Well.. after doing some tests I found that when pulling a multi-platform image, only the part of the image corresponding to the specific host's platform is pulled. The other platforms don't get pulled.

This means that the matrix approach wouldn't bring any improvements over the multi-platform docker image approach for the end user. Only (maybe) multi runner builds, which I don't believe is necessary for this project as the multi-image approach still builds in parallel pretty fast, as you can see in the build logs.

You can find the build logs of the latest test build here:
https://github.com/danieldabate/RTSPtoWeb/actions/runs/1875816937

And the multi-platform Docker image here: 
https://github.com/danieldabate/RTSPtoWeb/pkgs/container/rtsptoweb/15439851?tag=v2.2.0

If you do:
`docker pull ghcr.io/danieldabate/rtsptoweb:v2.2.0`
you will see that the hash of the pulled image is the same as if you pull only the piece of you host's platform _(you can find those under the **OS / Arch** tab)_.

Reply:
Thanks for the detail. It sounds like the only win could be that it runs the actions in parallel, but it just takes 3 minutes, so not too bad at all.

Image on a raspberry pi:
```
$ docker image ls ghcr.io/danieldabate/rtsptoweb
REPOSITORY                       TAG       IMAGE ID       CREATED       SIZE
ghcr.io/danieldabate/rtsptoweb   latest    f5d4e70cd318   5 hours ago   37.6MB
ghcr.io/danieldabate/rtsptoweb   v2.2.0    f5d4e70cd318   5 hours ago   37.6MB
```

Image on amd64:
```
$ docker image ls ghcr.io/danieldabate/rtsptoweb
REPOSITORY                       TAG       IMAGE ID       CREATED       SIZE
ghcr.io/danieldabate/rtsptoweb   latest    b7554063b244   5 hours ago   40.7MB
ghcr.io/danieldabate/rtsptoweb   v2.2.0    b7554063b244   5 hours ago   40.7MB
```

Great!